lib/clientv3/common_test.go

```diff
@@ -31,6 +31,8 @@ import (
 	"github.com/projectcalico/libcalico-go/lib/testutils"
 )
 
+var MatchResource = testutils.Resource
```

Comment:
Isn't this already defined in testutils?

Reply:
Yes, but that's in a different package (testutils, as opposed to clientv3_test).  The reason for it here is to avoid all the test code having to say `Expect(x).To(testutils.MatchResource(...))`.